src/core/markdown/ProseMirrorTransformer/getTransformers.ts

```diff
@@ -0,0 +1,19 @@
+import {transformEmptyParagraph} from './emptyRowParser';
+
+import {TransformFn} from '.';
+
+type GetTransformersProps = {
+    emptyRowTransformer?: boolean;
+};
+
+type GetPMTransformersType = (config: GetTransformersProps) => TransformFn[];
+
+export const getPMTransformers: GetPMTransformersType = ({emptyRowTransformer}) => {
+    const transformers = [];
+
+    if (emptyRowTransformer) {
+        transformers.push(transformEmptyParagraph);
```

Comment:
Add today: add a new method to the ExtensionBuilder

Reply:
> Add today: add a new method to the ExtensionBuilder

in the future